handler/engine/README.md

```diff
@@ -108,4 +118,31 @@ The images below illustrates the above rules.
 * `A` node can also respond with its custom transitions. Then we have to configure them in a graph. 
 Otherwise, if the `custom` transition is set but is not declared, then the `FAILURE` status is returned
 
-![Node with exits](assets/images/a_custom_no_configuration.png)
\ No newline at end of file
+![Node with exits](assets/images/a_custom_no_configuration.png)
+
+## Fragment's log
+A fragment's log contains details about task evaluation. When node processing ends (or 
+raises an exception), the engine appends the new [entry](https://github.com/Knotx/knotx-fragments/blob/master/handler/engine/src/main/java/io/knotx/fragments/engine/EventLogEntry.java) 
+in the fragment's log containing:
+- task name
+- node identifier
+- node status
+- [node log](#node-log)
+- transition
+- timestamp.
+
+Node status is a simple text value managed by the engine. It resembles a fragment's status but is a 
+bit more accurate (such as a `UNSUPPORTED_TRANSITION` value).
+
+Let's see the example fragment's log. There is a fragment that defines a task named `taskName`. The 
+task is a graph of two nodes: `A` and `B`.
+
+![A and B ends correctly](assets/images/a_success_b_success.png)
+
+The `A` node responds with the `_success` transition. Then the `B` node starts processing and responds 
```

Comment:
I think we're missing some punch line in this example, it could be more verbose + what is the outcome for the reader?

Reply:
I would suggest something during actions updates.